integrations/docusaurus/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "scripts": {
     "build": "shx rm -Rf ./dist && tsc --declaration && postcss src/theme.css -o dist/theme.css",
-    "dev:doesnt-work": "cd ./playground && pnpm dev",
+    "dev:doesnt-work": "cd ./playground && pnpm run dev",
```

Comment:
I added run command because otherwise `knip` would detect `dev` as binary.
I've done this a couple of times.